projects/cdk/tokens/active-element.ts

```diff
@@ -35,13 +35,15 @@ export const TUI_ACTIVE_ELEMENT = new InjectionToken<Observable<EventTarget | nu
 
             return merge(
                 focusout$.pipe(
-                    takeUntil(mousedown$),
-                    repeatWhen(() => mouseup$),
+                    repeatWhen(notifications =>
+                        notifications.pipe(switchMap(() => mouseup$)),
+                    ),
                     withLatestFrom(removedElement$),
                     filter(([event, removedElement]) =>
                         isValidFocusout(getActualTarget(event), removedElement),
                     ),
                     map(([{relatedTarget}]) => relatedTarget),
+                    takeUntil(mousedown$),
```

Comment:
I am not sure but this can be mistake

Reply:
No, this does not affect the order, also e2e test shows that there was misbehavior before

Reply:
reverted